class/actions_searcheverywhere.class.php

```diff
@@ -10,7 +10,7 @@ class ActionsSearcheverywhere
 	function printSearchForm($parameters, &$object, &$action, $hookmanager) {
 		global $langs,$db,$conf;
 
-		if (in_array('searchform',explode(':',$parameters['context'])) && (DOL_VERSION <= 3.8
+		if (in_array('searchform',explode(':',$parameters['context'])) && (floatval(DOL_VERSION) <= floatval('3.8')
```

Comment:
```suggestion
		if (in_array('searchform',explode(':',$parameters['context'])) && (floatval(DOL_VERSION) <= 3.8)
```

3.8 is already a float